server/game/cards/02-AoA/TheGreyRider.js

```diff
@@ -0,0 +1,49 @@
+const Card = require('../../Card.js');
+
+class TheGreyRider extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            optional: true,
```

Comment:
you can just pass `fight: true, reap: true` to this.play so you don't need to do the same thing three times